src/main/charts/bitbucket/values.yaml

```diff
@@ -752,6 +752,110 @@ bitbucket:
     #
     upstreamUrl:
 
+  # Mesh configuration
+  mesh:
+    # -- Enable Bitbucket Mesh. See: https://confluence.atlassian.com/bitbucketserver/bitbucket-mesh-1128304351.html
+    #
+    enabled: true
```

Comment:
Doesn't matter here, but we need to make a decision at some point about whether to enable this by default.